### PR TITLE
Fix tenant onboarding status stucked after stack created

### PR DESCRIPTION
**Pull request by hanrobyn**
----
1. After stacked created when onboarding,  the OnboardingService received an event with error Codepipeline's arn in GCR.
```
{
    "version": "0",
    "id": "be3a95aa-80a9-3830-8553-c7aef201cef7",
    "detail-type": "CodePipeline Pipeline Execution State Change",
    "source": "aws.codepipeline",
    "account": "111",
    "time": "2022-11-18T09:49:22Z",
    "region": "cn-north-1",
    "resources": [
        "arn:aws:codepipeline:cn-north-1:111:sb-test-tenant-e49fb688-main"
    ],
    ...
}
```
which causes the corresponding tenant has not created, and the onboarding status stays in "Provisioned".

2. Fix some url in GCR.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: client/web/src/dashboard/DashboardComponent.js [hunk 1]
```diff
@@ -263,7 +267,7 @@ export const DashboardComponent = (props) => {
                                 </ECRInstructions>
                               </dt>
                               <dd className="mb-3">
-                                {awsAccount}.dkr.ecr.{awsRegion}.amazonaws.com
+                                {awsAccount}.dkr.ecr.{awsRegion}.{serviceUrlSuffix}
```
Comment (PoeppingT) [inline client/web/src/dashboard/DashboardComponent.js:270]: It looks like you've replaced `amazonaws.com` with `amazon.com` in non-GCR regions.

**Review by PoeppingT — CHANGES_REQUESTED** [review]
Overall LGTM, thanks for the review! Unfortunate that CodePipeline sends erroneous events in GCR.

File: client/web/src/dashboard/DashboardComponent.js (hunk 1 above)
Comment (hanrobyn) [reply client/web/src/dashboard/DashboardComponent.js:270]: It's a mistake, fixed it.

**Review by PoeppingT — APPROVED** [review]
Looks good to me, thanks for the PR!

**Pull request merged**